frontend/src/containers/User/AccountSetting/AccountSetting.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import Button from "react-bootstrap/Button";
+
+import { userActions } from "../../../store/actions";
+import "./AccountSetting.css";
+
+class AccountSetting extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            description: this.props.myInfo.description,
+            email: this.props.myInfo.email,
+        };
+    }
+
+    componentDidMount() {
+        this.props.onGetMyInfo(this.props.myId);
+    }
+
+    clickApplyHandler = () => {
+        const newUserInfo = {
+            description: this.state.description,
+            email: this.state.email,
+        };
+        this.props.onEditMyInfo(newUserInfo);
+    }
+
+    render() {
+        return (
+            <div className="AccountSetting">
+                <div className="AccountSettingContent">
+                    <div className="EditDescArea">
+                        <h3 id="editDescTag">Your Description</h3>
+                        <textarea
+                          id="editDescription"
+                          rows="4"
+                          type="text"
+                          value={this.state.description}
+                          onChange={(event) => this.setState({ description: event.target.value })}
+                        />
+                    </div>
+                    <div className="EditEmailArea">
+                        <h3 id="editEmailTag">Your Email</h3>
+                        <input
+                          id="editEmail"
+                          type="text"
+                          value={this.state.email}
+                          onChange={(event) => this.setState({ email: event.target.value })}
+                        />
+                    </div>
+                    <div className="ButtonArea">
+                        <Button id="applyButton" onClick={this.clickApplyHandler}>Apply</Button>
+                    </div>
+                </div>
+            </div>
+        );
+    }
+}
+
+const mapStateToProps = (state) => ({
+    myId: state.auth.my.id,
+    myInfo: state.user.selectedUser,
```

Comment:
I think this part is unnecessary, PrivateRouter calls always API /api/user/me and sets state.auth.me on literally all pages.

Reply:
@njsh4261 I think you can refer to [this commit of my PR](https://github.com/swsnu/swpp2019-team3/pull/91/commits/103c758609ecaf955b8e75d9cf05c6afe78e7b1f) or `components/Header/Header.js` in the current master branch.